requirements/extra-ml-requirements.txt

```diff
@@ -3,8 +3,8 @@
 ##
 # Required by mlflow.azureml
 azureml-sdk==1.2.0
-# Required by mlflow.keras
-keras
+# Required by mlflow.tensorflow
+keras>=2.8.0
```

Comment:
Why 2.8.0?

Reply:
Can we just remove this and let tensorflow select the right keras version?

Reply:
typo, should use  >=2.6.0

Reply:
> Can we just remove this and let tensorflow select the right keras version?
Sounds a good idea.